### DISCUSSION TITLE
Images for kanban entries like trello has

### DISCUSSION THREAD

**Discussion by naxey** [mgmeyers/obsidian-kanban#844] [Ideas & Feature Requests]
<img width="273" alt="image" src="https://github.com/mgmeyers/obsidian-kanban/assets/40920984/6f8778e4-e5de-44b7-af71-e41639921306">

**Comment by TheGarageDragon** [comment]
I second this Idea, I think it would be great and add a lot of customisation options and readability

**Reply by TheGarageDragon** [reply]
An example of how it might help:
![SmartSelect_20240209_192133_Trello](https://github.com/mgmeyers/obsidian-kanban/assets/33641753/e912fe73-cd20-4bfa-8071-9e4f3e09ca17)
